apps/extension/src/core/domains/balances/store.BalanceTotals.ts

```diff
@@ -43,17 +43,16 @@ export const trackBalanceTotals = async () => {
         }, {} as Record<string, BalanceJson[]>)
 
         const totals = Object.fromEntries(
-          Object.keys(accounts).flatMap((address) =>
-            settings.selectableCurrencies.map((currency) => {
-              const balances = new Balances(balancesByAddress[address] ?? [], {
-                tokens,
-                tokenRates,
-              })
+          Object.keys(accounts).flatMap((address) => {
+            const balances = new Balances(balancesByAddress[address] ?? [], {
```

Comment:
Just moved this out of the for-loop for (premature) perf optimisation

Reply:
nice, thanks!